lib/Controller/WopiController.php

```diff
@@ -437,7 +439,14 @@ public function putFile($fileId,
 				$this->userSession->setUser($editor);
 			}
 
-			$file->putContent($content);
+			try {
+				$this->retryOperation(function () use ($file, $content){
+					return $file->putContent($content);
+				});
+			} catch (LockedException $e) {
+				$this->logger->logException($e);
+				return new JSONResponse(['message' => 'File locked'], Http::STATUS_INTERNAL_SERVER_ERROR);
```

Comment:
We should probably use STATUS_CONFLICT but right now Collabora behaves as if the file was saved successfully if that is returned, so let's keep the 500 status for now